docs/docs/learn/introduction/gateway/index.md

```diff
@@ -0,0 +1,52 @@
+
+## Next Steps for Gateway
+
+We are interested in working closely with builders looking to connect their smart contracts as new _strategies_ for BOB Gateway users.
+
+These are some of the features we're working on for Gateway's next upgrade, with new ideas being added frequently.
+
+- "Unwrap" your wrapped BTC on BOB by withdrawing back to BTC on Bitcoin mainnet
+- "Unstake" your BTC LST/LRT back to BTC on Bitcoin mainnet
+- Swapping from BTC to any ERC20 through a DEX (already possible, not fully implemented yet)
+
+:::info Integrate BOB Gateway Into your App
+[See our Builder Guide](/learn/builder-guides/gateway) to learn more about our SDK and extending BOB Gateway's functionality to bring Bitcoiners one transaction away from your use-case.
+:::
+
+## How Gateway Works
+
+1. Liquidity providers (LPs) temporarily lock wrapped Bitcoin (WBTC or tBTC) in escrow smart contracts on BOB.
+1. A user makes a request to the off-chain relayer to reserve some of the available liquidity.
+1. The user sends BTC to the liquidity provider's Bitcoin address. A hash of the user's order is included in the `OP_RETURN` of their transaction, including data such as the recipient's EVM address on BOB.
+1. The relayer trustlessly verifies the user's Bitcoin transaction by submitting a Merkle proof to an on-chain [Light Client](/learn/builder-guides/relay.md), granting the relayer permission to withdraw the LP's wrapped Bitcoin without needing to use an oracle.
+1. Gateway sends the LP's wrapped Bitcoin to the user's EVM address. If the user requested a Bitcoin LST/LRT, that token is minted using the LP's wrapped Bitcoin _before_ it is sent to the user.
+
+## Architecture
+
+![architecture](./architecture.png)
+
+### User Flow
+
+1. A user requests to swap BTC for wrapped BTC (e.g. WBTC, tBTC, or FBTC) or staked BTC (e.g. SolvBTC.BBN, uniBTC)
+1. The user gets a "quote" of which Gateway smart contract is an available route (i.e. which LP they can swap with)
+1. The user creates an "order" with the relayer to reserve the LP's liquidity
+1. The user creates a Bitcoin transaction, updating the order with their txid
+1. The relayer monitors the Bitcoin chain and sends the LP's wrapped BTC to the user after the txid seen on Bitcoin. Specifically, the relayer submits a Merkle proof of the Bitcoin tx to an [on-chain light client](/learn/builder-guides/relay) for trustless verification of the user's intent.
+
+### Liquidity Provider (LP) Flow
+
+1. An LP asks the relayer to deploy a new Gateway contract, which functions as an escrow for their funds. This is permissioned at the moment because BOB pays the transaction fees.
+2. The LP deposits wrapped Bitcoin (e.g. WBTC, tBTC, FBTC) in their Gateway contract.
+3. The LP can only withdraw their funds or update their swap fees after a delay so that the relayer has time to finish open orders. The relayer will not accept new orders during this delay until reset.
```

Comment:
Below this, I think we should have a trust assumptions section.

Gateway is currently operating with training wheels. These are going to be addressed and replaced as the product matures.

- Currently, there's a single relayer. The relayer pays the gas on behalf of the user so that users do not need to make a transaction on BOB to onboard to BOB. As malicious actors could craft malicious strategies, we are currently restricting adding strategies and decentralizing relayers until Gateway is more mature.
- LPs receive BTC directly on Bitcoin from the users. The off-chain relayer cannot interfere with the funds received to the LPs.
- The relayer needs to complete the intent of the user on BOB, i.e., submitting the Merkle proof with the intent on BOB to unlock wBTC/tBTC from the Gateway contracts. If the off-chain relayer is offline, user funds are stuck. Users cannot submit the proofs themselves at the moment. In the future, users will be able to submit their own BTC proofs when Gateway is switching to a full Bitcoin light client and adds functionality to protect LPs from possibly malicious strategy contracts.